build/Makefile

```diff
@@ -445,6 +445,14 @@ setup-prometheus:
 		$(HELM_ARGS) \
 		-f $(mount_path)/build/prometheus.yaml
 
+run-prometheus: $(ensure-build-image)
+	$(DOCKER_RUN) \
+		kubectl --namespace metrics port-forward $(shell kubectl get pods --namespace metrics -l 'app=prometheus,component=server' -o name) 9090
```

Comment:
Don't we have a portforward target already for Prometheus and Grafana ?

https://github.com/googleforgames/agones/blob/b134abbd4f2e4a9ab9fd57d3cadf19821e4fb7e3/build/README.md#make-grafana-portforward

Reply:
Ok, good. Didn't notice that.